src/main/java/teammates/common/util/ActivityLogEntry.java

```diff
@@ -1,631 +1,220 @@
-        if (timeTaken == null) {
-            return "";
-        }
-        
-        String colorCode = "";
-        if (timeTaken >= TIME_TAKEN_WARNING_LOWER_RANGE && timeTaken <= TIME_TAKEN_WARNING_UPPER_RANGE) {
-            colorCode = "warning";
-        } else if (timeTaken > TIME_TAKEN_WARNING_UPPER_RANGE && timeTaken <= TIME_TAKEN_DANGER_UPPER_RANGE) {
-            colorCode = "danger";
-        }
-        return colorCode;
-    }
-    
-    public String getLogEntryActionsButtonClass() {
-        
-        String className = "";
-        if (message.toLowerCase().contains(Const.ACTION_RESULT_FAILURE.toLowerCase())) {
-            className = "btn-warning";
-        } else if (message.toLowerCase().contains(Const.ACTION_RESULT_SYSTEM_ERROR_REPORT.toLowerCase())) {
-            className = "btn-danger";
-        } else {
-            className = "btn-info";
-        }
-        return className;
-    }
-
-    public String getUrlToShow() {
-        if (url.contains("user=")) {
-            return url;
-        }
-        // If not in masquerade mode, add masquerade mode
-        if (url.contains("?")) {
-            return url + "&user=" + googleId;
-        }
-        return url + "?user=" + googleId;
+    public String getActionUrl() {
+        return actionUrl;
     }
     
-    public String getId() {
-        return id;
+    public String getLogMessage() {
+        return logMessage;
     }
     
-    public void setKeyStringsToHighlight(String[] strings) {
-        this.keyStringsToHighlight = strings;
+    public long getLogTime() {
+        return logTime;
     }
     
-    public boolean toShow() {
-        return toShow;
+    public long getActionTimeTaken() {
+        return actionTimeTaken;
     }
     
-    public void setToShow(boolean toShow) {
-        this.toShow = toShow;
+    public String getActionServletName() {
+        return actionServletName;
     }
     
-    public long getTime() {
-        return time;
+    public String getActionName() {
+        return actionName;
     }
     
-    public String getServletName() {
-        return servletName;
+    public String getUserRole() {
+        return userRole;
     }
     
-    public String getAction() {
-        return action;
+    public boolean isMasqueradeUserRole() {
+        return isMasqueradeUserRole;
     }
     
-    public String getRole() {
-        return role;
+    public String getUserName() {
+        return userName;
     }
     
-    public String getName() {
-        return name;
+    public String getUserGoogleId() {
+        return userGoogleId;
     }
     
-    public String getGoogleId() {
-        return googleId;
+    public String getUserEmail() {
+        return userEmail;
     }
     
-    public String getEmail() {
-        return email;
+    public boolean isTestingData() {
+        return userEmail.endsWith(Const.ActivityLog.TESTING_DATA_EMAIL_POSTFIX);
     }
     
-    public Long getTimeTaken() {
+    public static class Builder {
```

Comment:
Yes, I mean to move the code position to the beginning of class ActivityLogEntry for viewer convenience, not out of the class ActivityLogEntry